.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
   - scripts/update_docs.sh
 
 after_success:
-  - tox -e coveralls
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Unless they have a "recommended best practice"

You should also remove the `coveralls` rule in `tox.ini`